pkg/services/authn/clients/password.go

```diff
@@ -12,9 +12,9 @@ import (
 )
 
 var (
-	errEmptyPassword       = errutil.NewBase(errutil.StatusBadRequest, "password-auth.empty", errutil.WithPublicMessage("Invalid username or password"))
-	errPasswordAuthFailed  = errutil.NewBase(errutil.StatusBadRequest, "password-auth.failed", errutil.WithPublicMessage("Invalid username or password"))
-	errInvalidPassword     = errutil.NewBase(errutil.StatusBadRequest, "password-auth.invalid", errutil.WithPublicMessage("Invalid password or username"))
+	errEmptyPassword       = errutil.NewBase(errutil.StatusUnauthorized, "password-auth.empty", errutil.WithPublicMessage("Invalid username or password"))
```

Comment:
were they not synced with context handler?

Reply:
or is it to fix enumeration?

Reply:
No, wrong status code

Reply:
I think there's an issue you can close if that's the case